src/Ocelot/Configuration/RateLimitMiddlewareType.cs

```diff
@@ -0,0 +1,7 @@
+namespace Ocelot.Configuration;
+
+public enum RateLimitMiddlewareType
+{
+    Ocelot,
+    DotNet,
```

Comment:
Can you clarify whether you are saying xml dev docs must be added or you are against of having them anywhere in the project? I ask, because it is not consistent throughout the project.

Reply:
Not an issue, because we've agreed to delete this `enum`. It is not required.